web/component2/src/demo_components/prop_validation.dart

```diff
@@ -33,11 +33,10 @@ class PropTypesTestComponent extends UiComponent2<PropTypesTestProps> {
   // This is closer to what it looks like in JS, but might be confusing syntax.
   @override
   get propTypes => {
-        getPropKey((props) => props.twoObjects, typedPropsFactory):
-            (props, propName, componentName, location, propFullName) {
+        getPropKey((p) => p.twoObjects): (props, info) {
           final length = props.twoObjects?.length;
           if (length != 2) {
-            return PropError.value(length, propName, 'must have a length of 2');
+            return PropError.value(length, info.propName, 'must have a length of 2');
           }
           return null;
```

Comment:
`return null` is no longer part of the boilerplate
```suggestion
```

Reply:
https://github.com/cleandart/react-dart/pull/228#discussion_r343752079